NamingSystems/ods-organization-code-namingsystem.xml

```diff
@@ -0,0 +1,36 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+  <id value="ods-organization-code-namingsystem" />
+  <name value="ODSOrganisationCode" />
+  <status value="draft" />
+  <kind value="identifier" />
+  <date value="2023-08-01" />
+  <publisher value="NHS England" />
+  <contact>
+        <name value="Organisation Data Service" />
```

Comment:
This needs a discussion. Ideally the IOPS team contact details should be here, or we have both contact details here.

Reply:
Agree, probably worth documenting that in UK Core and NHSe design approach for going forward. NamingSystems, removed, but have used this principle, and added the ODS contact details to their codesystems.